lib/JsonNode.cpp

```diff
@@ -21,6 +21,40 @@
 #include "JsonDetail.h"
 #include "StringConstants.h"
 
+namespace

```

Comment:
Empty namespace?

Reply:
_anonymous_ namespace, not empty. This is C++ way to create static (local to a file) functions.